Gemfile

```diff
@@ -37,9 +38,6 @@ gem 'uglifier', '>= 1.3.0'
 # Use CoffeeScript for .js.coffee assets and views
 gem 'coffee-rails'
 
-# See https://github.com/sstephenson/execjs#readme for more supported runtimes
-gem 'libv8', '>= 8.4.255.0' # Min version for Mac OS 10.15
```

Comment:
@barmintor I ran into issues with libv8 on M1 and tried removing it, and the app still runs -- locally, at least!  I'm not sure if your JS/webpack updates removed the need for it, or if it may still be needed for production.  We'll want to test out real server deployment before merging this PR.